homeassistant/components/plex/__init__.py

```diff
@@ -72,18 +73,21 @@
 _LOGGER = logging.getLogger(__package__)
 
 
-def setup(hass, config):
+async def async_setup(hass, config):
     """Set up the Plex component."""
-    hass.data.setdefault(PLEX_DOMAIN, {SERVERS: {}, DISPATCHERS: {}, WEBSOCKETS: {}})
+    hass.data.setdefault(
+        PLEX_DOMAIN,
+        {SERVERS: {}, DISPATCHERS: {}, WEBSOCKETS: {}, PLATFORMS_COMPLETED: {}},
+    )
 
     plex_config = config.get(PLEX_DOMAIN, {})
     if plex_config:
-        _setup_plex(hass, plex_config)
+        await _async_setup_plex(hass, plex_config)
 
     return True
 
 
-def _setup_plex(hass, config):
+async def _async_setup_plex(hass, config):
```

Comment:
We don't need to make this a coroutine, since we don't await inside.
```suggestion
def _async_setup_plex(hass, config):
```